sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -91,6 +112,18 @@ export class AppConfigurationClient {
     }
   }
 
+  /**
+   * Add a setting into the Azure App Configuration service, failing if it 
+   * already exists.
+   * 
+   * Example usage:
+   * ```ts
+   * const result = await client.addConfigurationSetting("MyKey", { label: "MyLabel", value: "MyValue" });
```

Comment:
Thanks for including in-line examples!

Reply:
The KeyVault API was my guide and it was a good teacher :)